### ISSUE TITLE
Incorrect typings for StateWithHistory

### ISSUE THREAD

**Issue by utkarshsaxena93** [omnidan/redux-undo#200]
Hello all,
I believe the typing for StateWithHistory [here](https://github.com/omnidan/redux-undo/blob/master/typings.d.ts#L4) is incorrect unless I am missing something about `redux-undo`.
With the current typing, I will have to specify my initial state as 
```
const INITIAL_STATE = {
    past: State[];
    present: State;
    future: State[];
    _latestUnfiltered: State;
    group: any;
    index: number;
    limit: number;
  }

// based on

export interface StateWithHistory<State> {
    past: State[];
    present: State;
    future: State[];
    _latestUnfiltered: State;
    group: any;
    index: number;
    limit: number;
  }
```
Where as what I want my initial state to look like is
```
const INITIAL_STATE = {
    past: State[];
    present: State;
    future: State[];
  }

// which should look like
 export interface StateWithHistory<State> {
    past: State[];
    present: State;
    future: State[];
    _latestUnfiltered?: State;
    group?: any;
    index?: number;
    limit?: number;
  }
```

Thoughts?

**Comment by omnidan** [comment]
@utkarshsaxena93 would you be up for sending a PR fixing the typing?

**Comment by nmay231** [comment]
Resolved in #248 :+1:
